alembic/versions/db981cb9dfed_movies_table.py

```diff
@@ -0,0 +1,33 @@
+"""movies_table
+
+Revision ID: db981cb9dfed
+Revises: 24dfa943a7d2
+Create Date: 2021-08-02 16:20:10.919320
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'db981cb9dfed'
+down_revision = '24dfa943a7d2'
+branch_labels = None
+depends_on = None
+
+tableName = "movies"
+
+def upgrade():
+    op.create_table(
+        tableName,
+        sa.MetaData(),
+        sa.Column("id", sa.String(255), primary_key=True), 
```

Comment:
As tu essayé de passer ta fonction dans une fonction lambda ? 
```suggestion
        sa.Column("id", sa.String(255), primary_key=True, default=lambda: str(uuid4())), 
```

Reply:
ça ne fonctionne pas mais j'ai bien essayé.